lib/decision_engine.rb

```diff
@@ -11,26 +11,25 @@ def result(board)
 
   def win?(board)
     p = board.positions
-    equal_not_empty?(p[0], p[1], p[2]) ||
-      equal_not_empty?(p[3], p[4], p[5]) ||
-      equal_not_empty?(p[6], p[7], p[8]) ||
-      equal_not_empty?(p[0], p[3], p[6]) ||
-      equal_not_empty?(p[1], p[4], p[7]) ||
-      equal_not_empty?(p[2], p[5], p[8]) ||
-      equal_not_empty?(p[0], p[4], p[8]) ||
-      equal_not_empty?(p[2], p[4], p[6])
+    winning_combination?(p[0], p[1], p[2]) ||
```

Comment:
Can this be refactored so it's easier to understand what these numbers are? Like maybe make a list of the numbers and map over them...or splat them or something?

Reply:
Yes. I'm saving that for a 4x4 / NxN board which is coming. Aware it looks a bit magic at the moment.